recipes/soxr/all/conanfile.py

```diff
@@ -27,73 +28,65 @@ class SoxrConan(ConanFile):
         "with_openmp": False,
         "with_lsr_bindings": True
     }
-    generators = "cmake"
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    exports_sources = "CMakeLists.txt"
```

Comment:
In general this file can be delete. when switching to the new toolchain + helper

Reply:
Removed it..